src/store/item-store.js

```diff
@@ -70,6 +70,28 @@ class ItemStore extends Store {
     }
 
 }
+function getItemToEdit(itemAddress){
+    let result = readAddress(itemAddress);
+    if (result.type !== 'item') {
+        throw new Error('Address is not an item.');
+    }
+    return items[result.index];
+}
+function changeTimestamp(itemAddress, date){
+    let itemToChange = getItemToEdit(itemAddress);
+    let timestamp = moment(date);
+    let hour = parseInt(timestamp.format('H'));
+    let minute = parseInt(timestamp.format('m'));
```

Comment:
NACK (?): Won't these times always be 0:00, because you're creating a `moment` object from a date (with no timestamp)?